dht/token-sharding.hh

```diff
@@ -45,10 +49,17 @@ public:
     sharder(unsigned shard_count = smp::count, unsigned sharding_ignore_msb_bits = 0);
     virtual ~sharder() = default;
     /**
-     * Calculates the shard that handles a particular token.
+     * Calculates the shard that handles a particular token for reads.
+     * Use shard_for_writes() to determine the set of shards that should receive writes.
      */
     virtual unsigned shard_of(const token& t) const;
```

Comment:
This should be renamed shard_for_reads() by the end of the series to prove all callers have been adjusted.

Perhaps we should rename it now, alias shard_of() to it, switch call sites, then remove shard_of() when done.

Reply:
Yes